k2/python/k2/fsa.py

```diff
@@ -252,36 +250,35 @@ def __setattr__(self, name: str, value: Any) -> None:
                 self.arcs.values()[:, -1] = _as_int(value.detach())
         else:
             assert name != 'properties'  # should be set by getter/setter or by
```

Comment:
assertion fails here

Reply:
@freewym, are you sure you're not using an older version of k2?  I fixed this by changing:
```
@property
 def properties(self, value):
```
to:
```
@properties.setter
 def properties(self, value):
```